schema.sql

```diff
@@ -1,5 +1,84 @@
 /* Database schema to keep the structure of entire database. */
-
+-- \c vet_clinic
+DROP TABLE IF EXISTS animals;
 CREATE TABLE animals (
-    name varchar(100)
+    id INT PRIMARY KEY NOT NULL,
+    name varchar(100) NOT NULL,
+    date_of_birth DATE NOT NULL,
+    escape_attempts INT,
+    neutered BOOLEAN,
+    weight DECIMAL
```

Comment:
- Kindly rename the `weight` column to `weight_kg` as the requirement says. 😀

Reply:
< Actioned 😃  />